trezorlib/tests/device_tests/test_msg_cardano_sign_transaction.py

```diff
@@ -19,22 +19,35 @@
 import pytest
 
 from trezorlib import cardano, messages
+from trezorlib.exceptions import TrezorFailure
 
 from .conftest import setup_client
 
-VECTORS = [
+PROTOCOL_MAGICS = {
+    'mainnet': 764824073,
+    'testnet': 1097911063
+}
+
+EXAMPLE_INPUTS = [
```

Comment:
maybe rename to INPUT_FIXTURES?

Reply:
perhaps SAMPLE_INPUTS, but "FIXTURES" does not sound too familiar